packages/frontend/src/stores/messagelist.ts

```diff
@@ -402,27 +402,24 @@ class MessageListStore extends Store<MessageListState> {
     jumpToMessage: this.scheduler.lockedQueuedEffect(
       'scroll',
       async (
-        msgId: number | undefined,
+        jumpToMessageId: number | undefined,
         highlight = true,
         addMessageIdToStack?: undefined | number
```

Comment:
Hmm, but that's even more confusing, in this use case there is an message.parentId passed as "msgId" and an message.id passed as argument msgParentId. 

So that's why I mean that some explaining documentation or could help understanding the code and thus also avoid bugs in the future